microcosm_pubsub/tests/test_consumer.py

```diff
@@ -100,3 +112,35 @@ def test_raw_consume():
         data="data",
         media_type=DerivedSchema.MEDIA_TYPE,
     ))))
+
+
+def test_json_consume():
+    """
+    Test that message sent as JSON could be delivered
+
+    """
+    graph = create_daemon_with_naive_cache()
```

Comment:
One small improvement you could make here is to pass the message in – it won't properly set the sqs_client, but it should still hit the `config.update` portion. Then you can test that the config for `sqs_envelope` and `sqs_consumer` are as expected.

Reply:
Thanks! Adding.

Reply:
It looks like adding such improvement requires adding changes to `microcosm_daemon` as well.
Creating a follow-up ticket for that.
https://globality.atlassian.net/browse/DEVOPS-1495